docs/topics/practices.rst

```diff
@@ -107,6 +106,33 @@ reactor after ``MySpider`` has finished running.
     runner = CrawlerRunner()
 
     d = runner.crawl(MySpider)
+    from twisted.internet import reactor
+
+    d.addBoth(lambda _: reactor.stop())
+    reactor.run()  # the script will block here until the crawling is finished
+
+Same example but using a non-default reactor, is only necessary call ``install_reactor`` if you are using ``CrawlerRunner`` since ``CrawlerProcess`` already does this automatically.
+
+.. code-block:: python
+
+    import scrapy
+    from scrapy.crawler import CrawlerRunner
+    from scrapy.utils.log import configure_logging
+
+
+    class MySpider(scrapy.Spider):
+        # Your spider definition
+        ...
+
+
+    configure_logging({"LOG_FORMAT": "%(levelname)s: %(message)s"})
+    from scrapy.utils.reactor import install_reactor
+
+    install_reactor("twisted.internet.asyncioreactor.AsyncioSelectorReactor")
```

Comment:
Can you please check if this example works when `AsyncioSelectorReactor` is not added to settings via settings.py/custom_settings? Not sure if the reactor is compared to the setting in this case.

Reply:
Is not compared.

https://github.com/scrapy/scrapy/blob/631fc65fadb874629787ae5f7fdd876b9ec96a29/scrapy/crawler.py#L117-L127

Logs:
````
2024-05-22 20:48:25 [asyncio] DEBUG: Using selector: KqueueSelector
````